backend/Models/Data/Sighting.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace WhaleSpotting.Models.Data;
+
+public class Sighting
+{ 
+    [Key]
+    public int Id { get; set; }
+    public int UserId { get; set; }
+    public int WhaleSpeciesId { get; set; }
+    public float Lattitude { get; set; }
+    public float Longitude { get; set; }
+    public int AreaNameId { get; set; }
```

Comment:
The Area table is not going to be our mvp so may not be made for a while. I think rather than having this column exist when the table doesn't exist yet we should remove from this model (and migration) I've added a note to the ticket that implements that foreign key that the field will need to be created on this table